product-development/README.md

```diff
@@ -0,0 +1,89 @@
+
+After the epic is completed, the engineer should find the next epic to work on, either as a DRI owner, or as a collaborator. Engineers should self-organize and coordinate timelines ad hoc when picking epics. There are no stable teams, no timeboxed sprints, and no synced sprints across teams. Engineers should be the DRI for at most one epic at a given time. But an engineer may be contributing to multiple epics concurrently.
+
+### Epic owner and epic states
+- An epic is in one of three states: `Unstarted`, `In Progress`, `Closed`. 
+- If an epic is in `In Progress` or `Closed`, it must have exactly one owner, who is the DRI.
+
+### DRI responsibilties and non-responsibilities as epic owner
+
+| What | Reponsibilities | Non-responsibilities | Why |
+| --- | --- | --- | --- |
+| Communication and status | The DRI is responsible for maintaining the epic state, description, and attached stories updated as the single source of truth of overall work status, at least on a daily basis.<br/><br/>The acceptance criteria sections are _especially critical_, including presenting the problem and final implementation in a company meeting.| The DRI is not responsible for communicating epic work status, beyond keeping the epic itself updated as the single source of truth. The DRI is not responsible for sharing work on Slack, meetings, or keeping Tilters otherwise updated. The DRI _can_ provide additional communication, but is not required to do so. Victor will help with broader general communication needs. | This encourages anyone who wants to learn about and collaborate on the epic to visit the epic itself (and it's attached stories) to see the latest updates, allowing the DRI to focus on actual work and leverage a single place for providing updates. |
+| Decision-making | The DRI is responsible for driving the epic to completion in a timely fashion within the target **_7 days_**.<br/><br/>The DRI epic owner is responsible for reducing scope (deferring valuable work to future epics) if additional work is discovered. | The DRI is not responsible for seeking full consensus or permission from all Tilters.<br/><br/>The DRI is not responsible for waiting for a Tilter's response even if they asked for one, prior to making a decision. | We are biasing toward shipping in small increments quickly. So even if a sub-optimal change was shipped, this is preferable to _not_ shipping at all or shipping slowly, because we want to move quickly with small changes. The risk of doing something very bad is small since the scope is small, freeing up the DRI to have more confidence in their decision making, since the downside is low. If subsequent epics are needed to further tweak a shipped change, this is an indication of success, not failure. |
```

Comment:
Thanks @landism . I think I was trying to do too much in this section, and your commentary rightly pokes a lot of holes in it. Rather than try to solve/address all the points you raise one by one, I simplified the section to just a few lines for the purpose of this PR (which is to update the high-level planning and execution process). 

(I think your points do deserve discussion, but perhaps in a later PR / forum.)

Do you mind  reviewing the latest updates in this section and let me know if it still doesn't look right to you?